burotel/indico_burotel/plugin.py

```diff
@@ -94,3 +97,6 @@ def _before_request(self):
             if 'reason' not in request.json:
                 request._cached_json['reason'] = 'Burotel booking'
                 request.data = request._cached_data = json.dumps(request._cached_json)
+
+    def _override_rb_email_templates(self, *args, **kwargs):
```

Comment:
`_override_templates` since technically this signal isn't specific to the email templates?